articles/azure/azs-how-configure-mars-backups-powershell.md

```diff
@@ -0,0 +1,91 @@
+---
+title: How to backup file(s) and folder(s) on Azure Stack VMs using Microsoft Azure recovery services (MARS) - PowerShell | UKCloud Ltd
```

Comment:
remove the parentheses and make the title shorter:
How to back up files and folders on Azure Stack VMs using PowerShell